commerce-service/service.xml

```diff
@@ -249,7 +249,7 @@
 		<!-- Other fields -->
 
 		<column name="commerceAccountId" type="long" />
-		<column name="commerceCurrencyId" type="long" />
+		<column name="commerceCurrencyCode" type="String" />
```

Comment:
this should be sorted below `shippingAddressId` now (at least that low -- put it where you want it after that)